### ISSUE TITLE
Maybe a little more Fortran-friendly?

### ISSUE THREAD

**Issue by urbanjost** [mvertes/txt2man#31]
Great Utility. I had to tweek it a little to work with Fortran documenation
where in the SYNTAX section a Fortran procedure usually looks like
   function NAME(
or
   subroutine NAME(
instead of just
   NAME(

It would be nice if your original was more Fortran-friendly instead of primarily
easy to use for C-like syntax so I could just use it directly.

See the results so far at
   [Fortran Intrinsics](https://github.com/urbanjost/fortran-intrinsic-manpages)

Have your ever thought of adding an option to also generate HTML? You are already
doing the parsing and I find that I very often am converting my manpages to HTML
using manserver(1) or groff(1) which usually ends up in rather poor-quality
HTML. The *roff directives you are generating have 1-1 equivalents in HTML2 so it
seems like a natural and useful extension to your utility.

Thanks again for the great tool!

**Comment by mvertes** [comment]
I agree that adding support to Fortran would be nice and is certainly achievable. I will look at it.

I've done a variant of txt2man (unpublished) to generate HTML some time ago, so yes, this is something I could consider.

**Comment by urbanjost** [comment]
Great. I appreciate you taking it under consideration, that is all I can ask  Again, thanks for the great utility.
